index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <link rel="stylesheet" href="./assets/css/style.css" />
+        <title>Elizabeth's Portfolio Page</title>
+    </head>
+  <body>
+      <!--navigation bar of the web page-->
+      <header class="header">
+      <h1>Elizabeth's Portfolio</h1>
+      <nav class=nav-container> 
+      <div>
+        <button type="email"> <img src="images/designes/EmailIcon.jpg" class="email"> Email</button>
+        <a href="Name"> 
+      </div>
+      <div>
+        <button type="GitHub"> <img src="images/designes/GitHubIcon.gif" class="github">Git Hub</button>
+        <a href="GitHub"> 
+      </div>
+      <div>
+        <button type="linkedIn"> <img src="images/designes/LinkedInIcon.png" class="linkedin"> LinkedIn</button>
+        <a href="Link 3"> 
+      </div>
+      <!--dive that consist a little logo image -->
+      <div>
+          <div class="logo-box">
+          <a href="Logo">
+              <img src="images/designes/OIP.jfif" class="pink-heart-image"/>
+      </a>
+      </div>
+      </header>
+
+      <section class='main-section'>
+        <h2>About Me</h2>
+        <p>I enjoy taking complex problems and turning them into simple and beautiful interface designs. 
+          I also love the logic and structure of coding. I am currently studying at the University of Birmingham in the UK</p>
+          </section> 
+
+    <main>
+      <section class="champion">
+        <section class="content">
+        <section class="champion-image-container">
+          <img src="images/designes/10665682_10201529829726157_1848441216688015788_n.jpg">
+      </section>
+       
+    <section>
+      <h2>Project1</h2>
```

Comment:
I would move this `<h2>Project1</h2>` to inside the `<div class="row">` so that it is uniform with the rest of the section.